code/_core/obj/structure/interactive/nature/snow.dm

```diff
@@ -132,4 +136,6 @@ obj/structure/scenery/snow_bush/New()
 
 /obj/structure/interactive/tree/jungle_large/New(var/desired_loc)
 	icon_state = "tree[rand(1,6)]"
+	pixel_x = rand(-49,-46)
+	pixel_y = rand(-16,-14)
```

Comment:
this could probably be put on the /obj/structure/interactive/tree interactive structure instead.